src/web/lib/components/App/index.js

```diff
@@ -26,6 +26,7 @@ const mapStateToProps = state => {
     {}
   );
   return {
+    isFirefox: navigator.userAgent.toLowerCase().includes('firefox/'),
```

Comment:
Do we want to differentiate Fennec and iOS here?

Reply:
Hmm, probably - though now that you mention it, I don't think we want to just display "Download firefox" for those browsers. Maybe we need a third message that pushes folks to desktop Firefox in the mobile case.